### ISSUE TITLE
Anemone's `strtod` still not on par with Double parser in Attoparsec

### ISSUE THREAD

**Issue by erikd-ambiata** [ambiata/anemone#48]
Looking again at using Anemone in Walrus to parse `Double` values and find that `stdtod` fails on 
strings like`1.e1`and `12.` where there the `.` character is either at the end of the string or just preceding the exponent.

**Comment by amosr** [comment]
Yeah, I guess we should aim to match whatever libc `strtod` does. I wonder whether there's a standard for what is and is not valid.
